workflows/qc/quality-check-standard.wdl

```diff
@@ -119,12 +120,29 @@ workflow quality_check {
     call ngsderive.read_length as ngsderive_read_length { input: bam=quickcheck.checked_bam, bam_index=bam_index, max_retries=max_retries }
     call ngsderive.encoding as ngsderive_encoding { input: ngs_files=[quickcheck.checked_bam], prefix=prefix, max_retries=max_retries }
 
-    call picard.bam_to_fastq { input: bam=quickcheck.checked_bam, max_retries=max_retries }
```

Comment:
Why the switch from Picard to samtools?

Reply:
This switch was inspired by problems with `SJBALL033086_G1.WholeGenome` in the alpha test cohort. Unfortunately the QC results didn't elucidate much details as to why, but `Picard SamToFastq` took over a week to convert this BAM. That same BAM was processed by `samtools collate`->`samtools fastq` in under 24 hours.

I haven't investigated the "average case", but I'm pretty sure it's saves time and memory in all cases. That's really an unchecked assumption on my part, as in the name of freeing storage I `rm`d all my test logs from before this switch. That was shortsighted of me.

But regardless of previous performance on "average cases", we have an example of runtime dropping from more than a week to under 36 hours with this switch. That's more than enough to justify it if you ask me.

As for using this approach everywhere? I'd definitely consider it. There are 2 downsides:
1) it complicates things by replacing one call with 2
2) it is a pretty significant bump in storage. Collated BAMs are really beefy and need to be stored.

I just got an idea that could solve both those problems though. One task which calls both tools in the same command block. By default `rm`s the collated BAM after FastQ conversion. Can be kept with a Boolean (which we'd need for QC to pass to Qualimap). It's only a partial solution to storage though, as the very large BAM still has to be created. So the storage capacity has to exist, it's just not used for long.